landing-page/content/services/row-level-deletes.html

```diff
@@ -1,9 +1,11 @@
 ---
-Title: Schema Evolution
-AsciinemaCast: schema_evolution.cast
+Title: Row-level Deletes
+Description: Iceberg supports the copy-on-write and merge-on-read approaches to handle row-level delete operations.
```

Comment:
The term "row-level deletes" probably doesn't connect with people coming to this page. Instead, I'd highlight "Expressive SQL changes" or something similar. What users care about is that they can use `MERGE INTO`, `UPDATE`, and `DELETE FROM`. (Mostly `MERGE` and `DELETE`)

So the description can be something like `Iceberg supports flexible SQL commands to merge new data, update existing rows, and perform targeted deletes. Iceberg can eagerly rewrite data files for read performance, or it can use delete deltas for faster updates.`

Reply:
I like it, updated!